### PR TITLE
[v1] Manually migrate to assimp 5.2.5 

### PR DESCRIPTION
**Pull request by traversaro**
I am not sure why, but even if the `v1` branch is listed in the abi migration branches, there was no assimp 5.2.5 migration. So with this PR I manually migrate the repo, to unblock https://github.com/conda-forge/hpp-pinocchio-feedstock/pull/18 and the rest of the assimp 5.2.5 migraiton.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by traversaro** [comment]
@conda-forge-admin, please rerender

**Review by jcarpent — APPROVED** [review]

**Comment by jcarpent** [comment]
We need to patch hpp-fcl 1.8.1 with updated cmake routines to find ASSIMP 5.2.5

**Review by ymontmarin — APPROVED** [review]

**Comment by ymontmarin** [comment]
Hi @traversaro and thank you for the great work ! I can deal with downstream `hpp-*` that requires explicitelly assimp after this merge !

**Comment by traversaro** [comment]
> Hi @traversaro and thank you for the great work ! I can deal with downstream `hpp-*` that requires explicitelly assimp after this merge !

Hopefully the downstream packages should be handled by the migration once we unblock hpp-pinocchio, but thanks in any case!

**Pull request merged**
